.ci/cabal.project.local

```diff
@@ -10,7 +10,7 @@ package *
 
 package clash-prelude
   ghc-options: -Werror
-  flags: +doctests +multiple-hidden
+  flags: +doctests +multiple-hidden -workaround-heisenbug
```

Comment:
Don't you want `+workaround-heisenbug`?

Reply:
No, that would set it for all GHC versions. `.ci/setup.sh` turns it on just for GHC 9.0.2.

Reply:
I like that `workaround-ghc19421` is more specific then `workaround-heisenbug`.
But I still hate it, the name is meaningless to my mind.
And also confusingly if one were to read the [GHC-9.0.2 release notes](https://downloads.haskell.org/ghc/9.0.2/docs/html/users_guide/9.0.2-notes.html), it lists 19421 as fixed.

I propose something like `workaround-ghc-mmap-crash`

Reply:
Pretty strong word for such a minor thing 😅.

Suggestion seems fine to me. Approved the PR too.